ansible/roles/uan_interfaces/tasks/main.yml

```diff
@@ -343,3 +412,20 @@
 
   when:
     - ('Compute' not in group_names) or ('Compute' in group_names and uan_user_access_cfg == 'CHN')
+
+- name: "DBG: Get uan_interfaces exit time"
+  local_action:
+    module: shell
+    cmd: 'date "+%T"'
+  register: end_time
+
+- name: "DBG: Performance times"
```

Comment:
I don't think all of the debug statements are necessary. With ARA, we can get a breakdown of the time spent in each task and a summary of the whole playbook.

![image](https://github.com/Cray-HPE/uan/assets/81820210/3bc8f888-eb4b-43fd-8001-048b6cb86e30)

Reply:
Removed debug statements and we'll use ARA to get performance data.